src/main/splitting/tasks_tests_handler.py

```diff
@@ -0,0 +1,290 @@
+    if len(out_files_list) != len(in_files_list):
+        raise ValueError('Length of out files list does not equal in files list')
+    in_and_out_pairs = __separate_in_and_out_files_on_pairs(in_files_list, out_files_list)
+    return in_and_out_pairs
+
+
+def __get_out_file_by_in_file(in_file: str):
+    return 'out_' + str(in_file.split('.')[0].split('_')[-1]) + '.txt'
+
+
+def __separate_in_and_out_files_on_pairs(in_files: list, out_files: list):
+    pairs = []
+    for in_file in in_files:
+        out_file = __get_out_file_by_in_file(in_file)
+        if out_file not in out_files:
+            raise ValueError('List of out files does not contain a file for ' + in_file)
+        pairs.append((in_file, out_file))
+    return pairs
+
+
+def __get_java_class(source_code: str):
+    class_key_word = 'class'
+    rows = source_code.split('\n')
+    for row in rows:
+        if class_key_word in row:
+            class_index = row.index(class_key_word)
+            return row[class_index + len(class_key_word) + 1:].replace(' ', '').replace('{', '')
+    return SOURCE_OBJECT_NAME
+
+
+# Wrap all values from input in the print command
+def __create_py_input_file(txt_in_file: str, task: str, file_name=INPUT_FILE_NAME):
+    code = ''
+    with open(__get_task_file(txt_in_file, task), 'r') as f:
+        for line in f:
+            code += 'print("' + line.strip('\n') + '")' + '\n'
+    create_file(code, get_extension_by_language(LANGUAGE.PYTHON.value), __get_task_file(file_name, task))
+
+
+# For python scripts it is an in file with extension py and for other cases, it is an in file with extension txt
+def __get_in_file_for_current_test(cur_in_file: str, task: str, language=LANGUAGE.PYTHON.value):
+    if language == LANGUAGE.PYTHON.value:
+        __create_py_input_file(cur_in_file, task)
+        return INPUT_FILE_NAME + '.' + get_extension_by_language(language)
+    return cur_in_file
+
+
+def __remove_compiled_files():
+    remove_directory(__get_source_folder())
+    create_directory(__get_source_folder())
+
+
+def __run_python_test(in_file: str, expected_out: str, task: str, source_file_name=SOURCE_OBJECT_NAME):
+    p1 = Popen(['python3', __get_task_file(in_file, task)], stdout=PIPE)
+    p2 = Popen(['python3', __get_compiled_file(source_file_name) + '.'
+                + get_extension_by_language(LANGUAGE.PYTHON.value)],
+               stdin=p1.stdout,
+               stdout=PIPE)
+    p1.stdout.close()
+    try:
+        signal.alarm(consts.MAX_SECONDS_TO_WAIT_TEST)
+        out, err = p2.communicate()
+        actual_out = out.decode("utf-8").rstrip("\n")
+        log.info("In-file: " + in_file + ", task: " + task + ", expected out: " + expected_out + ", actual out: " + actual_out)
+        return actual_out == expected_out
+    except TimeoutException:
+        log.info("In-file: " + in_file + ", task: " + task + ", Time is out")
+        return False
+    except:
```

Comment:
Please fix it. It is a bad way - to use only 'except' without a type of error. You can read about it here: https://stackoverflow.com/questions/21553327/why-is-except-pass-a-bad-programming-practice/21553515. If you want to catch any "safe" exceptions, please use 'except Exception', see more: https://stackoverflow.com/questions/730764/how-to-properly-ignore-exceptions